monkey/infection_monkey/exploit/tools/brute_force_exploiter.py

```diff
@@ -100,7 +100,7 @@ def _exploit(self, exploit_client: IRemoteAccessClient, host: TargetHost, interr
         credential_combinations = self._get_credentials()
 
         for brute_force_credentials in interruptible_iter(credential_combinations, interrupt):
-            tags: MutableSet[str] = set()
+            tags: Set[str] = set()
```

Comment:
don't use mutable arguments as defaults! https://towardsdatascience.com/python-pitfall-mutable-default-arguments-9385e8265422

Reply:
It seems this has already been changed to a set. Not sure why github doesn't show it as "outdated".

Reply:
My bad, the mutable arguments are actually in `_publish_exploitation_event` and `_publish_propagation_event`